core/app/services/spree/import_service/error.rb

```
@@ -0,0 +1,20 @@
+module Spree
+  module ImportService
+    class Error < StandardError
+      attr_reader :message, :identifier
+
+      def initialize(identifier:, message:)
+        @identifier = identifier
+        @message = message
+      end
```

RuboCop (1.75.5)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)